harvestingkit/inspire_cds_package/from_inspire.py

```diff
@@ -443,17 +443,17 @@ def update_collaboration(self):
 
     def update_542(self):
         """Change 542__e -> 542__3 and 542__e:Article to 542__3:publication."""
-        for field in record_get_field_instances(self.record, '542'):
+        fields = record_get_field_instances(self.record, '542')
+        for field_idx, field in enumerate(fields):
```

Comment:
doesn't look like `field_idx` is used anywhere, so why enumerate?

Reply:
Sorry that was a leftover, I will fix it! The reason for this change is coming from the fact that before the transformation was like:
```
<datafield tag="542" ind1=" " ind2=" ">
     <subfield code="g">Another Field</subfield>
     <subfield code="e">Article</subfield>
     <subfield code="a">Another Field</subfield>
</datafield>
```
would result to:

```
<datafield tag="542" ind1=" " ind2=" ">
    <subfield code="g">Another Field</subfield>
     <subfield code="a">Another Field</subfield>
</datafield>
<datafield tag="542" ind1=" " ind2=" ">
         <subfield code="3">Publication</subfield>
</datafield>
```
instead of:

```
<datafield tag="542" ind1=" " ind2=" ">
     <subfield code="g">Another Field</subfield>
     <subfield code="3">Publication</subfield>
     <subfield code="a">Another Field</subfield>
</datafield>
```